engine/gameobject/src/gameobject/comp_script.cpp

```diff
@@ -223,11 +224,14 @@ namespace dmGameObject
         RunScriptParams run_params;
         run_params.m_UpdateContext = params.m_UpdateContext;
         CompScriptWorld* script_world = (CompScriptWorld*)params.m_World;
+        update_result.m_TransformsUpdated = false;
         uint32_t size = script_world->m_Instances.Size();
         for (uint32_t i = 0; i < size; ++i)
         {
             HScriptInstance script_instance = script_world->m_Instances[i];
             if (script_instance->m_Update) {
+                // TODO: Find out if the scripts actually sent any transform events
+                update_result.m_TransformsUpdated = true;
```

Comment:
if update function called - then it makes sense to mark it as transform updated (original TODO still actual)